### ISSUE TITLE
Screen Reader Accessibility of Embedded Youtube Videos

### ISSUE THREAD

**Issue by brndnb** [FreeCodeCamp/freecodecamp#28]
There isn't a play button accessible to screen readers in embedded YouTube videos. This means that the previous live pairing videos are not accessible. I tested it out with Voice Over and can't access a play button at all.

The workaround for this is to offer a direct link to the YouTube videos, because the YouTube website itself is accessible to screen readers. I suggest we do that. Or, if nothing else, we need more copy on the live pairing page to explain what is there. Right now there isn't enough context for what is on the page if a person can't see it.

Note: Vimeo videos have an accessible play button, so challenge videos are okay.

**Comment by QuincyLarson** [comment]
Which operating system/browser are you using? These are iframe embeds straight from Youtube, so this is a Youtube problem. Do you adblock or any chrome extensions active? Try opening it in incognito mode. If you can't get it to work, we should include a link below the iframe embed, but I suspect a small minority of users are effected by this.

**Comment by brndnb** [comment]
It's a known youtube problem, but I think a link below the iframe would do a lot for the minority of users that experience this frustration on a regular basis.

I was just listening to a podcast that mentioned this issue, so that's why I tried it to confirm.

**Comment by QuincyLarson** [comment]
I've added links below each archived youtube video.
